day_7/ceasar_cipher_attempt.rb

```diff
@@ -0,0 +1,41 @@
+class CeasarCipher
+
+  def initialize
+  end
+
+  def encode(word, shift_num)
+    #make aplphabet into array
+    alphabet = 'abcdefghijklmnopqrstuvwxyz'.split('')
+    #make encoded alphabet array
+    encoded_alphabet = alphabet.unshift(alphabet.last(shift_num + 1)).flatten(1)
+
+    #turn word to be ciphered into array
+    word_array = word.split('')
+    letter_index = []
+
+     word_array.each do |letter|
+      #for each letter, return the corresponding index in original alphabet array.
+      letter_index << alphabet.index(letter)
+    end
+
+    encoded_array = []
+
+    letter_index.each do |index|
+      encoded_array << encoded_alphabet[index]
+    end
+
+    puts encoded_array.join('')
+  end
+end
+
+cipher = CeasarCipher.new
+
+cipher.encode('abcde', 2)
+
+  # alphabet.each do |letter|
+  #   alphabet_index.push(alphabet.index(letter))
+  # end
+  #
+  # cipher_word.push(alphabet[(alphabet.index(letter) + shift_num)])
+
+  # encoded_array.push(encoded_alphabet[num])
```

Comment:
Right now, when I run this, the shift goes backwards, so an 'a' shifted 2 becomes a 'y'. Traditionally, they shift forward, but totally developers decision! An interesting extension could be make a ruby program that grabs some user input to define the shift, as well as whether it shifts forwards or backwards. This is above and beyond here, but an idea of something to play around with as you continue to experiment.